common/cluster/metadata.go

```diff
@@ -59,6 +59,8 @@ type (
 		IsGlobalNamespaceEnabled() bool
 		// IsMasterCluster whether current cluster is master cluster
 		IsMasterCluster() bool
+		// GetClusterID return the cluster ID, which is also the initial failover version
+		GetClusterID() int64
```

Comment:
@yiminc @yycptt @yux0 another solution is to use the UUID from cluster metadata table, what do you think?

Reply:
I think UUID is overkill. Initial failover should be good.